fbgemm_gpu/codegen/embedding_backward_split_template.cu

```diff
@@ -963,7 +968,6 @@ split_embedding{{ "_nobag" if nobag else "" }}_backward_codegen_{{ optimizer }}_
             // over 48 KB per block are architecture-specific, as such they
             // must use dynamic shared memory (rather than statically sized
             // arrays) and require an explicit opt-in using cudaFuncSetAttribute()".
-
```

Comment:
Would you mind removing such unnecessary changes that are left in the code? It makes us easy to review the code (also keep the diff minimized).

Reply:
The empty line is taken back.